README.md

```diff
@@ -53,7 +53,7 @@ So it must be installed on the system to compile and use those crates.
 ### [PROJ] for binaries
 
 Using the [`proj` crate] requires some system-dependencies installation.\
-The version `7.1.0` of [PROJ] is needed (used and tested by maintainers).
+The version `7.2.1` of [PROJ] is needed (used and tested by maintainers).
```

Comment:
This is more a self-reminder : We have to be consistent about the version all over the codebase (dockerfile, makefile, CI). So there is a bit more work to be done on our side, or we may stay on `7.1.0` everywhere (let us know if you noticed some things required from `7.2`).

Reply:
I'm unsure if there are any relevant behavioral changes. 

I updated this naively, just to conform to the version of proj that the crate expects: https://github.com/georust/proj/blob/0.22.0/proj-sys/build.rs#L11